app/src/main/res/values-en/strings.xml

```diff
@@ -1365,19 +1365,19 @@ Otherwise, you can download another keyboard in the app store. Popular keyboards
     <string name="street_parking_parallel">"Parallel"</string>
     <string name="street_parking_diagonal">"Diagonal"</string>
     <string name="street_parking_perpendicular">"Perpendicular"</string>
-    <string name="street_parking_on_street">"On roadway"</string>
-    <string name="street_parking_half_on_kerb">"Partially on roadway"</string>
-    <string name="street_parking_on_kerb">"Adjacent to roadway"</string>
-    <string name="street_parking_street_side">"Adjacent to roadway only in dedicated spaces"</string>
-    <string name="street_parking_painted_area_only">"On roadway only in dedicated spaces"</string>
+    <string name="street_parking_on_street">"On-street with no curb"</string>
+    <string name="street_parking_half_on_kerb">"On-street but partially in the road"</string>
```

Comment:
"in the road"? Typo?

Reply:
What's wrong with "in the road"?  You're along the curb, part in the shoulder and part in the road?

Reply:
because it sounds like it is "in" the road, as in embedded in the asphalt. I'd have expected "on"

Reply:
Oh, it's "in the road" in the same sense as "in the way".  "on the road" would be a bit weird here since you could be part _in_ the shoulder and part _in_ the roadway but still **on** the road.  Sorry, English is funny.

Reply:
I can totally see why this is insane for a non-native speaker 😂

Reply:
ok